governance/polls/Vault Compensation Denomination - April 13, 2020.md

```diff
@@ -38,11 +38,11 @@ At the time of liquidation, 10 ETH sold at 90 DAI/ETH would have yielded 900 DAI
 There are two possible scenarios:
 
 1. **If this poll decides that the compensation shall be in ETH**
-   - This vault holder could expect a compensation of at most 1.3 ETH
+   - This vault holder could expect a compensation scaled by 1.3 ETH
```

Comment:
What do you mean by *"scaled"*? I think I understand what it means for DAI but uncertain about ETH.

Also:
> will have appreciated since

Do you mean "will raise in value"?
> could be adjusted to approximate the amount of appreciation

Do you mean "can be adjusted to reflect current value"?
I suggest to use more simple wording.

Reply:
> What do you mean by "scaled"?

_Scaled_ means to multiply by some number.

Regarding your other comments, sure, go ahead and change it as you suggest.

Reply:
I meant do you want to say something like this or I miss the idea?
> a compensation scaled by 1.3 ETH

"a compensation of 1.3 ETH with an additional haircut %"

Reply:
It is called a _haircut_ if we multiply by a number <1. I guess the term _scaling_ does not imply whether the number is <1 or >1.

Reply:
Exactly. And I wonder if there's a case when we have to scale ETH compensation up? I there's no one the term "scale" is probably a bad choice.

Reply:
If the estimated compensation comes in very low then maybe we'll want to scale ETH compensation up? I don't want to preclude things that could happen in theory.

Reply:
I think this is not worded very well, I was puzzled when I read this and I've read over this document many times over the last few days.

My suggestion to make it more clear:

```suggestion
   - This vault holder could expect a compensation of 1.3 ETH (or a scaled multiple of it, e.g. 1.1x)
```